app/assets/stylesheets/styles/post-edit.scss

```diff
@@ -10,16 +10,16 @@ label {
   margin: 0;
 }
 
-#container {
+.editContainer {
   display: grid;
   grid-template-columns: 12.5% 75% 12.5%;
 }
 
-#header-container {
+.editHeaderContainer {
   margin-top: 10px;
```

Comment:
This kinda goes for the whole file but we should avoid using absolute units for spacing